src/saml2/response.py

```diff
@@ -409,11 +409,17 @@ def _verify(self):
                 raise RequestVersionTooHigh()
 
         if self.asynchop:
-            if (
-                self.response.destination
-                and self.response.destination not in self.return_addrs
+            if not (
+                 getattr(self.response, 'destination')
```

Comment:
1. no brackets necessary.
2. `getattr()` raises `AttributeError` if no third parameter is given.

Reply:
done here
https://github.com/peppelinux/pysaml2/commit/1476eb93c622b82d866464c156ab66ae0c059b83

thank you, I completely agree